File: Udemy/Algoritmos e Logica de Programacao/Java/ExerciciosJavaEstruturaRepetitiva/src/application/ProblemaFatorial.java
package application;

import java.util.Locale;
import java.util.Scanner;

public class ProblemaFatorial {

	public static void main(String[] args) {
		/*
		 * Problema "fatorial" (adaptado de URI 1153)
Fazer um programa para ler um número natural N (valor máximo: 15), e depois calcular e mostrar o
fatorial de N.
		 * */
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		System.out.print("Digite o valor de N: ");
		int n = sc.nextInt();
		int fat = 1;
		for(int i = n;i>0;i--) {
			fat = fat * i;
		}
		
		System.out.println("FATORIAL = "+fat);

		sc.close();

	}

}

File: Algoritmos e Logica de Programacao/C/Estrutura Repetitiva/ProblemaFatorial.c
/*
Problema "fatorial" (adaptado de URI 1153)
Fazer um programa para ler um número natural N (valor máximo: 15), e depois calcular e mostrar o
fatorial de N.
*/

#include<stdio.h>
#include<stdlib.h>
int main (){

	int i, n, fat;
	
	fat = 1;

	printf("Digite o valor de N: ");
	scanf("%d",&n);
	for(i=n;i>0;i--){
		fat = fat * i;
	}
	printf("FATORIAL = %d",fat);

	
	

    return 0;
}
